fmriprep/interfaces/confounds.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+
+import os
+import shutil
+import numpy as np
+import pandas as pd
+from niworkflows.nipype import logging
+from niworkflows.nipype.interfaces.base import (
+    traits, TraitedSpec, BaseInterfaceInputSpec, File, Directory
+)
+from niworkflows.interfaces.base import SimpleInterface
+
+LOGGER = logging.getLogger('interface')
+
+
+class GatherConfoundsInputSpec(BaseInterfaceInputSpec):
+    signals = File(exists=True, mandatory=True, desc='input signals')
+    dvars = File(exists=True, mandatory=True, desc='file containing DVARS')
+    fd = File(exists=True, mandatory=True, desc='input framewise displacement')
+    tcompcor = File(exists=True, mandatory=True, desc='input tCompCorr')
+    acompcor = File(exists=True, mandatory=True, desc='input aCompCorr')
+    cos_basis = File(exists=True, mandatory=True, desc='input cosine basis')
+    motion = File(exists=True, mandatory=True, desc='input motion parameters')
+    aroma = File(exists=True, mandatory=True, desc='input ICA-AROMA')
```

Comment:
These are not mandatory as presently written. I believe the idea is that the individual inputs can fail or (in the case of `aroma`, at least) not be run, and the interface will still work.